invisible_cities/cities/buffy.py

```diff
@@ -62,7 +62,13 @@ def buffy( files_in          : OneOrManyFiles
 
     max_time          = check_max_time(max_time, buffer_length)
     npmt, nsipm       = get_n_sensors(detector_db, run_number)
-    pmt_wid, sipm_wid = pmt_and_sipm_bin_width_safe_(files_in)
+    try:
+        pmt_wid, sipm_wid = pmt_and_sipm_bin_width_safe_(files_in)
+    except tb.exceptions.NoSuchNodeError:
+        return dict(events_in   = 0,
+                    events_resp = 0,
+                    evtnum_list = [])
```

Comment:
Ok, what does that mean?

Reply:
All cities return a `Namespace` (which you can import from `argparse`). Here you are returning a `dict`, which breaks that pattern. I suggest returning a `Namespace`, for which you only need to replace `dict` -> `Namespace`, because they have a similar constructor.